plugins/container/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v0.6.0
+
+* [`05d713c`](https://github.com/falcosecurity/plugins/commit/05d713c) docs(plugins/container): update changelog for `0.6.0`
+
+* [`b956a46`](https://github.com/falcosecurity/plugins/commit/b956a46) chore(plugins/container): bump to `0.6.0`
+
+* [`c8a839a`](https://github.com/falcosecurity/plugins/commit/c8a839a) feat(container): add basic exposed fields
+
```

Comment:
We should include https://github.com/falcosecurity/plugins/pull/1112

Reply:
I manually added it.